xbmc/GUIInfoManager.cpp

```diff
@@ -4276,16 +4276,6 @@ const infomap playlist[] =       {{ "length",           PLAYLIST_LENGTH },
 ///                  _string_,
 ///     Backend number
 ///   }
-///   \table_row3{   <b>`Pvr.HasEpg`</b>,
-///                  \anchor Pvr_HasEpg
-///                  _boolean_,
-///     Returns true when an epg is available.
-///   }
-///   \table_row3{   <b>`Pvr.HasTxt`</b>,
```

Comment:
Is this info no more used and working (Seen that Estuary not use it)?

Was thought from me to enable Teletext GUI button on xml only if Teletext is present.